app/src/main/java/org/p2p/wallet/home/ui/main/empty/epoxy/EmptyHomePopularTokenModel.kt

```diff
@@ -0,0 +1,77 @@
+package org.p2p.wallet.home.ui.main.empty.epoxy
```

Comment:
you can understand which implementation you are viewing now by looking at the package name